types/stripe-v3/index.d.ts

```diff
@@ -44,27 +44,47 @@ declare namespace stripe {
         retrievePaymentIntent(
             clientSecret: string,
         ): Promise<PaymentIntentResponse>;
+        /** @deprecated */
         handleCardPayment(
             clientSecret: string,
             element: elements.Element,
             options?: HandleCardPaymentOptions,
         ): Promise<PaymentIntentResponse>;
+        /** @deprecated */
         handleCardPayment(
             clientSecret: string,
             options?: HandleCardPaymentWithoutElementsOptions,
         ): Promise<PaymentIntentResponse>;
+        confirmCardPayment(
+            clientSecret: string,
+            data?: ConfirmCardPaymentOptions,
+        ): Promise<PaymentIntentResponse>;
```

Comment:
Perhaps the [argument `options`](https://stripe.com/docs/js/payment_intents/confirm_card_payment#stripe_confirm_card_payment-options) was missed?

Reply:
It was indeed, thanks for seeing that @demensky.

It's fixed.